RetailCoder.VBE/Refactorings/ImplementInterface/ImplementInterfaceRefactoring.cs

```diff
@@ -70,10 +70,11 @@ private void ImplementMissingMembers()
             var implementedMembers = GetImplementedMembers();
 
             var nonImplementedMembers =
-                interfaceMembers.Where(
-                    d =>
-                        !implementedMembers.Select(s => s.IdentifierName)
-                            .Contains(_targetInterface.ComponentName + "_" + d.IdentifierName)).ToList();
+                interfaceMembers.Where(d => !implementedMembers.Select(s => s.IdentifierName)
+                                    .Contains(_targetInterface.ComponentName + "_" + d.IdentifierName))
```

Comment:
I'm going to nitpick here, but this could use a named method. We need to get into a habit of breaking down our LINQ stuff into more descriptive code.